backend/api/models.py

```
@@ -1,3 +1,120 @@
+
+# Company Model
+class Company(models.Model):
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE)
+    company_name = models.CharField(max_length=255)
+    created_at = models.DateTimeField(default=timezone.now)
+
+    def __str__(self):
+        return self.company_name
+
+# Payment Model
+class Payment(models.Model):
```

Ruff (0.8.2)

Model does not define `__str__` method

(DJ008)